pkg/controllers/cluster/sync.go

```diff
@@ -53,6 +54,12 @@ func (cc *ClusterReconciler) sync(c *scyllav1.ScyllaCluster) error {
 		return errors.Wrap(err, "failed to sync headless service")
 	}
 
+	// Sync Cluster Pod Disruption Budget
+	if err := cc.syncPodDisruptionBudget(ctx, c); err != nil {
+		cc.Recorder.Event(c, corev1.EventTypeWarning, naming.ErrSyncFailed, MessagePodDisruptionBudgetSyncFailed)
```

Comment:
preexisting pattern (but sync failures (retriable) don't usually manifest into events, degraded condition on the status would be more fitting I guess)

Reply:
Lets fix it once for all failures as a follow up in 1.2: https://github.com/scylladb/scylla-operator/issues/513